### PR TITLE
add content managable about page

### PR DESCRIPTION
**Pull request by shakisparki**
add about page with markdown configurable contents

### CODE REVIEW

File: src/Directory/Data/Constants/ConfigKey.cs [hunk 1]
```diff
@@ -45,5 +45,9 @@ public static class ConfigKey
         public const string EthicsFieldIsCheckbox = "site.sampleresource.ethics.type";
         public const string SampleResourceName = "site.sampleresource.name";
 
+        // Editable About Page
+        public const string AboutpageTitle = "site.aboutpage.title";
```
Comment (beforan) [inline src/Directory/Data/Constants/ConfigKey.cs:49]: does the title get used anywhere?

File: src/Directory/Biobanks.Web/Biobanks.Web.csproj [hunk 2]
```diff
@@ -448,6 +449,7 @@
     <Content Include="Content\DataTables\css\common.scss" />
     <Content Include="App_Config\footer.json" />
     <Content Include="App_Config\header.json" />
+    <Content Include="Content\wordpress.json" />
```
Comment (beforan) [inline src/Directory/Biobanks.Web/Biobanks.Web.csproj:452]: what's this?

File: src/Directory/Data/Constants/ConfigKey.cs (hunk 1 above)
Comment (shakisparki) [reply src/Directory/Data/Constants/ConfigKey.cs:49]: absolutely forgot to take that out

File: src/Directory/Data/Migrations/Configuration.cs [hunk 3]
```diff
@@ -254,6 +254,15 @@ protected override void Seed(BiobanksDbContext context)
                             Description = "Enable/Disable Biobank approval to join a Network",
                             ReadOnly = false
                         },
+
+                        // Aboutpage Config
+                        new Config {
+                            Key = "site.aboutpage.bodytext",
+                            Value = @"##About Us
+The Foundation for Innovative New Diagnostics(FIND), a non - profit global health organization, supports activities and projects to improve diagnosis of infectious diseases in low - and middle - income countries(LMICs).FIND, acting at arm's length to offer visibility to existing biological resources for a wider audience, has created the FIND Virtual Biobank Directory (VBD). Funded by Unitaid, the VBD is a directory of searchable information on collections hosted by FIND but also those hosted by other organizations or networks worldwide. The VBD is an open-access, free of charge tool, which allows users and sample owners to interact and discuss without FIND�s intervention, unless explicitly invited to do so by either party.
```
Comment (beforan) [inline src/Directory/Data/Migrations/Configuration.cs:262]: this is environment specific content, it shouldn't be the default.

File: src/Directory/Biobanks.Web/Biobanks.Web.csproj (hunk 2 above)
Comment (shakisparki) [reply src/Directory/Biobanks.Web/Biobanks.Web.csproj:452]: that should not be there

**Review by beforan — CHANGES_REQUESTED** [review]

**Comment by beforan** [comment]
is this functionality supposed to be able to be turned off (e.g. TDCC doesn't use it today)?

**Comment by shakisparki** [comment]
> is this functionality supposed to be able to be turned off (e.g. TDCC doesn't use it today)?
@blargly ?
Is this something for now or for the extra work to be done later?

**Review by beforan — APPROVED** [review]

**Review by blargly — APPROVED** [review]

**Pull request merged**